src/KerosApp.php

```diff
@@ -43,6 +44,11 @@ public function __construct()
                 $this->get('/{id:[0-9]+}', CatController::class . ':getCat');
                 $this->post("", CatController::class . ':createCat');
             });
+            $this->group('/core/gender', function () {
```

Comment:
Ca devrait être 
$this->group("/core") et à l'intérieur du group "core", ajouter les deux get ("/gender" et "/gender/{id:[0-9]+}, ou encore un group "gender" dans un group "core"